tsconfig.json

```diff
@@ -1,17 +1,16 @@
 {
     "compilerOptions": {
-        "target": "es2015",
+        "target": "ESNext",
```

Comment:
You should stick to `es2016` or `es2017` (since the code relies on ES7 features) to ensure that no new ECMAScript features make it into the built code without a planned upgrade.